src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.vb

```diff
@@ -313,6 +393,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     ' NOTE: Not actually a keyword, but it's not worth introducing a new kind just for this.
                     builder.Add(New SymbolDisplayPart(SymbolDisplayPartKind.Keyword, Nothing, "AnonymousType"))
                     AddSpace()
+                ElseIf symbol.IsTupleType Then
+                    builder.Add(New SymbolDisplayPart(SymbolDisplayPartKind.AnonymousTypeIndicator, Nothing, "Tuple"))
```

Comment:
changed to AnonymousTypeIndicator. It seems it has been introduced later and VB was not updated to use it.

---
In reply to: [71446991](https://github.com/dotnet/roslyn/pull/12596#discussion_r71446991) [](ancestors = 71446991)